roles/openshift_node/defaults/main.yml

```diff
@@ -75,7 +98,7 @@ openshift_node_dnsmasq_except_interfaces:
 r_openshift_node_firewall_enabled: "{{ os_firewall_enabled | default(True) }}"
 r_openshift_node_use_firewalld: "{{ os_firewall_use_firewalld | default(False) }}"
 
-l_is_node_system_container: "{{ (openshift_use_node_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
+l_is_node_system_container: "{{ openshift_use_node_system_container | default(openshift_use_system_containers | default(openshift_is_containerized | default(false))) | bool }}"
```

Comment:
This is not correct.  Just because something is a container doesn't mean it's a system container.

We don't want to use system containers here because they will override /etc/systemd/system/origin-node.service whenever they are installed or updated, which is currently what is breaking me.

Reply:
We are completely removing containerized mode, so everything is either a system container or kubernetes.  Containerized now means only one thing - use containerized kubelet.  We could change the meaning of both, but they are gone.